quarkchain/cluster/master.py

```diff
@@ -1017,13 +1033,15 @@ def handle_new_root_block_header(self, header, peer):
         All update root block should be done in serial to avoid inconsistent global root block state.
         """
         self.root_state.validate_block(r_block)  # throw exception if failed
```

Comment:
Looks like validate_block could be removed (since add_block will do the same thing)?